.github/workflows/genie-performance-test.yml

```diff
@@ -0,0 +1,44 @@
+name: genie-performance-test
+
+on:
+  schedule:
+    - cron:  '0 8 * * *'
+
+jobs:
+  probe-genie-endpoint:
+    
+    name: Genie private endpoint probe
+
+    runs-on: ubuntu-latest
+    
+    env:
+      SLACK_URL_SECRET: ${{ secrets.SLACK_URL_SECRET }}
+      SLACK_CHANNEL: backend-performance-status 
+
+    steps:
+    - name: GET request to genie private page
+      env:
+        GENIE_DATA_ACCESS_TOKEN_SECRET: ${{ secrets.GENIE_DATA_ACCESS_TOKEN_SECRET }}
+        MAX_WAIT_TIME_IN_SECONDS: 300
+        URL: "https://genie-private.cbioportal.org/api/mutated-genes/fetch"
+        OK_RESPONSE_CODE: 200
+      # GET request to URL with a MAX_WAIT_TIME_IN_SECONDS second limit
+      #   if curl fails, automatically exit with curl's non-zero exit code
+      #   else if we did not get a OK_RESPONSE_CODE exit with a non-zero exit code
+      #   else we are OK!
+      run: |
+        curl --max-time $MAX_WAIT_TIME_IN_SECONDS -s -o /dev/null -w "%{http_code}" -X POST -H "Authorization:Bearer $GENIE_DATA_ACCESS_TOKEN_SECRET" "$URL" -H "accept:application/json" -H "Content-Type:application/json" -d "{\"studyIds\":[\"genie_private\"]}" > curl.out
```

Comment:
@mandawilson Nice! Would it be possible to also log the time the response took somewhere? Maybe it can be sent both to stdout and to the slack channel? That way even if it maybe doesn't go over the threshold we would have some idea of whether response time has changed over time.